packages/components/src/components/vertical-tabs/vertical-tab-content-container/vertical-tab-content-container.jsx

```diff
@@ -26,24 +25,32 @@ class VerticalTabContentContainer extends React.PureComponent {
                     >
                         {
                             action_bar.map(({ component, icon, onClick }, idx) => {
-                                const Component = component;
-                                return (
-                                    component ?
+                                if (component) {
+                                    const Component = component;
+                                    return (
                                         <Component key={idx} />
-                                        :
-                                        <div
-                                            id={`dt_${id}_close_icon`}
-                                            className='vertical-tab__action-bar-wrapper'
-                                            key={idx}
-                                            onClick={onClick}
-                                        >
-                                            <Icon
-                                                className='vertical-tab__action-bar--icon'
-                                                icon={icon}
-                                            />
-                                        </div>
+                                    );
+                                }
```

Comment:
While you are refactoring, do you think it is better to check the negation first, and then render without if statement? Something like: 

```jsx 
if (!component) return null;
const Component = component;
return (...)
```

Reply:
But according to the code we're still rendering something if `!component`. Perhaps I'm misunderstanding you?